schema/me/conversation.js

```diff
@@ -1,11 +1,157 @@
+      value: "other",
+    },
+    BLANK: {
+      value: "",
+    },
+  },
+})
 
-import { ConversationType } from "./conversations"
+export const MessageType = new GraphQLObjectType({
+  name: "MessageType",
+  description: "A message in a conversation.",
+  fields: {
+    id: {
+      description: "Impulse id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    from_email_address: {
+      description: "Email address of sender.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    snippet: {
+      description: "A snippet of the full message",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+})
 
-const { IMPULSE_APPLICATION_ID } = process.env
+export const ConversationParticipantType = new GraphQLObjectType({
+  name: "ConversationParticipantType",
+  description: "A participant in a conversation.",
+  fields: {
+    id: {
+      description: "Impulse id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    type: {
+      description: "The type of participant, e.g. Partner or User",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    email: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+})
+
+export const ConversationFields = {
+  id: {
+    description: "Impulse id.",
+    type: GraphQLString,
+  },
+  inquiry_id: {
+    description: "Gravity inquiry id.",
+    type: GraphQLString,
+  },
+  from: {
+    description: "The participant who initiated the conversation",
+    type: new GraphQLNonNull(ConversationParticipantType),
+    resolve: conversation => {
+      return {
+        id: conversation.from_id,
+        type: conversation.from_type,
+        name: conversation.from_name,
+        email: conversation.from_email,
+      }
+    },
+  },
+  to: {
+    description: "The participant responding to the conversation",
+    type: new GraphQLNonNull(ConversationParticipantType),
+    resolve: conversation => {
+      return {
+        id: conversation.to_id,
+        type: conversation.to_type,
+        name: conversation.to_name,
+        email: conversation.to_email,
+      }
+    },
+  },
+  buyer_outcome: {
+    type: GraphQLString,
+  },
+  buyer_outcome_at: date,
+  created_at: date,
+  purchase_request: {
+    type: GraphQLBoolean,
+  },
+  initial_message: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  last_message: {
+    type: new GraphQLNonNull(GraphQLString),
+    resolve: conversation => {
+      return get(conversation, "_embedded.last_message.snippet")
+    },
+  },
+  last_message_at: date,
+
+  artworks: {
+    type: new GraphQLList(ArtworkType),
+    resolve: (conversation, options, request, { rootValue: { artworksLoader } }) => {
+      let ids = []
+
+      for (const item of conversation.items) {
+        if (item.item_type === "Artwork") {
+          ids.push(item.item_id)
+        }
+      }
+
+      // ids = ["593ef5cd9c18db4f97a9efb7", "5406f53772616943601a0a00"]
```

Comment:
If this is commented out, and the `ids` that I've verified are correct go into the loader, I just get an empty array back.